src/scss/grommet-core/_objects.box.scss

```diff
@@ -8,6 +8,28 @@
 
   @include pad();
 
+  @include media-query(palm) {
```

Comment:
I suspect that in order to maintain good design balance, we might also need to scale the vertical and between padding as well, and also look into other components that do padding and margins, such as Form, Heading, and Headline.